int32 nForcePlates
forcePlate[] forcePlates